adoc/admin-gpus.adoc

```diff
@@ -0,0 +1,163 @@
+[[gpus]]
+= NVIDIA GPUs
+
+SUSE's CaaS Platform supports scheduling GPU-dependent workloads on NVIDIA GPUs as a technical preview. This can be a powerful way to run compute-intensive workloads such as machine learning pipelines. This section illustrates how to prepare your host machine to expose GPU devices to your containers, and how to configure Kubernetes to schedule GPU-dependent workloads.
+
+== Prepare the host machine
+
+=== Install the GPU drivers
+
+Not every worker node in the cluster need have a GPU device present. On the nodes that do have one or more NVIDIA GPUs, install the drivers from NVIDIA's repository.
+
+[source,bash]
+----
+# zypper addrepo --refresh https://download.nvidia.com/suse/sle15sp2/ nvidia
+# zypper refresh
+# zypper install x11-video-nvidiaG05
+----
+
+[NOTE]
+====
+For most modern NVIDIA GPUs, the G05 driver will support your device. Some older devices may require the x11-video-nvidiaG04 driver package instead. Check NVIDIA's documentation for your GPU device model.
+====
+
+=== Install the OCI hooks
+
+OCI hooks are a way for vendors or projects to inject executable actions into the lifecycle of a container managed by the container runtime (runc). SUSE provides an OCI hook for NVIDIA that enable the container runtime and therefor the kubelet and the Kubernetes scheduler to query the host system for the presence of a GPU device and access it directly. Install the hook on the worker nodes with GPUs:
+
+[source,bash]
+----
+# zypper install nvidia-container-toolkit
+----
+
+=== Test a GPU container image
+
+At this point, you should be able to run a container image that requires a GPU and directly access the device from the running container, for example using podman:
+
+[source,bash]
+----
+# podman run docker.io/nvidia/cuda nvidia-smi
```

Comment:
The container source for this test container is docker, do we need to be concerned with recommending using containers not packaged by us?

Reply:
while I *think* we can make this container. I don't see any reason too, it would bring our customers no value and we are not experts in supporting cuda in a container. I also don't think we are supporting this container, merely giving people the option to test their deployment with a tool provided by their GPU manufacturer.

Reply:
I agree with @Klaven. We can leave this container reference here.